webapp/plugins/index.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+// EXPERIMENTAL - SUBJECT TO CHANGE
+
+import store from 'stores/redux_store.jsx';
+
+global.window.plugins = {};
```

Comment:
Both global and window are unnecessary here. I think they refer to the same object when compiled down